CMakeLists.txt

```diff
@@ -67,6 +68,17 @@ if (DMLC_USE_SANITIZER)
   enable_sanitizers("${DMLC_ENABLED_SANITIZERS}")
 endif (DMLC_USE_SANITIZER)
 
+if(USE_PARQUET)
+  find_package(Arrow REQUIRED)
+  find_package(Parquet REQUIRED)
+  if(BUILD_SHARED_LIBS)
+    target_link_libraries(dmlc PRIVATE arrow_shared parquet_shared)
+  else()
+    target_link_libraries(dmlc PUBLIC arrow_shared parquet_shared)
```

Comment:
Do we need to change `arrow_shared` and `parquet_shared` into `arrow_static` and `parquet_static`, as there is a [static correspondence](https://arrow.apache.org/docs/cpp/cmake.html)?

Reply:
That would be great!